recipes/stumpy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "stumpy" %}
+{% set version = "1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ce2f46e7645d2a61d9a4f11a957b3b9bd833986ced44a9563332edc77229685c
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
```

Comment:
```suggestion
    - python >= 3.5
```

Reply:
Done

Reply:
There should only be a space before the version constraint:
```yaml
    - python >=3.5
```
Using either of these causes feedstock creation to fail:
```yaml
    - python >= 3.5
    - python>=3.5
```
(I've fixed it in master so there's not need to do anything, just letting you know for future)